src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -126,7 +126,8 @@ public static void waitTillConnectionDataLoadingCompletes(long timeoutInSeconds)
    * Example : For GCS pass actual bucket name as parameter.
    */
   private static void selectConnectionDataRow(String dataRow) {
-    waitTillConnectionDataLoadingCompletes(ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+   // waitTillConnectionDataLoadingCompletes(ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+    WaitHelper.waitForElementToBeClickable(CdfConnectionLocators.searchDirectoryInput, 120);
```

Comment:
Please use a named constant instead of 120. 
Also, please remove the commented line.

Reply:
Preferably use `ConstantsUtil.DEFAULT_TIMEOUT_SECONDS` which is 180.

Reply:
done.